engine/engine/celery.py

```diff
@@ -16,12 +16,9 @@
 
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "settings.prod")
 
-from django.db import connection  # noqa: E402
-
 logger = get_task_logger(__name__)
 logger.setLevel(logging.DEBUG)
 
-connection.cursor()
```

Comment:
had to remove this because the API container's startup command imports this file somewhere along the way, and `connection.cursor()` throws an exception if the database is unavailable